examples/c/publish_subscribe_cross_language/README.md

```diff
@@ -0,0 +1,110 @@
+# Cross-language Publish-Subscribe
+
+Before proceeding, all dependencies need to be installed. You can find
+instructions in the [C Examples Readme](../README.md).
+
+## Running The Example
+
+This example illustrates a robust publisher-subscriber communication pattern
+between a C publisher and a Rust subcriber application. The C publisher
```

Comment:
What do you think of not specifying here the language and state: between publisher and subscribers written in any language iceoryx2 supports. Then we can use the sentence in every other example and it makes it clear that also future examples in python, go or zig are meant as well

This goes also for the other remarks and sentences in the example. Just speak of subscribers or publishers.